### ISSUE TITLE
Using ansible-nas without the "nas" / share creation

### ISSUE THREAD

**Issue by point-source** [davestephens/ansible-nas#731]
**Is your feature request related to a problem? Please describe.**
I am looking for a project to help streamline the deployment of many of these media server / htpc apps (primarily the downloading-focused apps). This appears to be one of the most active and well-documented of the ones I've found. Unfortunately, I haven't been able to tell if this can be used easily in a configuration like mine where I have a dedicated NAS and a dedicated application/service server. I mount the NAS via NFS to the VMs on my application server.

**Describe the solution you'd like**
I would like to be able to tell ansible-nas how to access my existing file storage (via NFS mount) and then have it skip the deployment/creation of any storage or shares of its own. It should continue to perform all other responsibilities like docker configuration but just point everything at my existing storage.

**Describe alternatives you've considered**
Mounting my storage on the system/VM that I will target with ansible-nas and letting it make a share on top of a share (this feels very wrong)

**Additional context**
Maybe ansible-nas can already do what I'm looking for and I just haven't figured that out yet. In that case, I'd like to request better documentation that supports this use-case. Fwiw, I am very new to ansible and I recognize that if I had more experience, I could likely answer this question by digging through the configuration files. I am going to attempt to do this anyway but I wanted to leave a record of this request in case it results in a quicker answer and a better reference for those who follow.

Thanks for this awesome project!

**Comment by point-source** [comment]
Answering my own question here, this appears to be possible by appending `--skip-tags "nfs,samba"` to the ansible-playbook command like so: `ansible-playbook -i inventories/my-nas/inventory nas.yml --skip-tags "nfs,samba"`

This results in ansible skipping the installation of the samba and nfs role but it still attempts to use the `samba_shares_root` var to mount inside the various containers. As far as I can tell, this is the behavior I want since I can now manually mount an external share at the path that I will point `samba_shares_root` at.

I will leave this open for now, pending review from Dave just in case I am making an assumption that might tragically break things down the road. Based on my limited review of the code though, it doesn't seem that any of the containers are mounting the share via samba. They all appear to bind mount the local path so I think I'm in the clear. Please confirm.

**Comment by darshitpp** [comment]
@point-source you are correct, mounting isn't through samba, it uses the local file systems
